extensions/cloud/portability-cloud-google/src/main/java/org/dataportabilityproject/cloud/google/GoogleCloudModule.java

```diff
@@ -52,6 +55,12 @@
   // The path where Kubernetes stores Secrets.
   private static final String KUBERNETES_SECRETS_PATH_ROOT = "/var/secrets/";
 
+  private final ExtensionContext context;
+
+  GoogleCloudModule(ExtensionContext context) {
+    this.context = context;
```

Comment:
Maybe as this evolves, we can pass the objects provided by the context to another Constructor so we can test directly

GoogleCloudModule(HttpTransport transport, JSONFactor factory) {
 this.transport = transport;
 this.factory = factory;
}

Reply:
I had it that way first actually, I didn't want the params list to get too long as it's manually called, I'm fine with either way though. I'll leave for now and we can always change it later